pkg/operator/ceph/cluster/osd/envs.go

```diff
@@ -146,6 +147,10 @@ func pvcBackedOSDEnvVar(pvcBacked string) v1.EnvVar {
 	return v1.EnvVar{Name: PVCBackedOSDVarName, Value: pvcBacked}
 }
 
+func wipeDevicesFromOtherClustersEnvVar() v1.EnvVar {
+	return v1.EnvVar{Name: wipeDevicesFromOtherClustersVarName, Value: "true"}
```

Comment:
```suggestion
	return v1.EnvVar{Name: wipeDevicesFromOtherClustersVarName, Value: "true"}
```
We want to set it `true` by default in the env?

Reply:
No. By default we don't want to set this env. But if `spec.cleanup.wipdevicesFromOtherCluster` is set to `true`, then we want to set this env as `true`.